release_docs/INSTALL_CMake.txt

```diff
@@ -531,12 +531,18 @@ These five steps are described in detail below.
             set (ZLIB_USE_LOCALCONTENT ON CACHE BOOL "Use local file for ZLIB FetchContent" FORCE)
```

Comment:
Why is zlib not updated to v1.3.1 on the fly?

Reply:
We want to use the latest release after testing.

Reply:
OK. Why was libaec updated in this PR? Seems unrelated to zlib-ng.

Reply:
Because I have a bad habit of adding something I think is related.